pytest/tests/mocknet/mirror.py

```diff
@@ -219,6 +219,13 @@ def new_test(args, traffic_generator, nodes):
             args.genesis_protocol_version,
             genesis_time=genesis_time), all_nodes)
 
+    if args.stateless_setup:
+        logger.info('enabling in-memory trie in config')
+        pmap(lambda node: do_update_config(node, 'store.load_mem_tries_for_tracked_shards=true'), all_nodes)
```

Comment:
can you also make this change: https://docs.google.com/document/d/1pfMAHdsIYRtijmV3mPlvlNFUXptr5ylRV1npvUJhN58/edit#bookmark=id.zgdmugf7nhc9

pmap(lambda node: do_update_config(node, 'save_latest_witnesses=true'), all_nodes)